tests/data/argoverse/train1/273c1883-673a-36bf-b124-88311b1a80be/poses/city_SE3_egovehicle_315975643412234000.json

```diff
@@ -0,0 +1 @@
+{"rotation": [-0.8553418671199027, -0.0007711494300289546, -0.01385911873546937, -0.5178779977062857], "translation": [-273.2461280730776, 3052.5016282247057, -19.100182530254255]}
```

Comment:
why did these argoverse files change?

Reply:
@ayushbaid I added these tests before we implemented the ArgoverseLoader, so I generated some intermediate files for the FolderLoader back then. No I am just using the native dataset format in the ArgoverseLoader.